packages/esm-patient-chart-app/src/visit/visit-form/visit-date-time.component.tsx

```diff
@@ -107,6 +113,17 @@ const VisitDateTimeField: React.FC<VisitDateTimeFieldProps> = ({
           />
         </ResponsiveWrapper>
       </div>
+      {errors[dateFieldName]?.message && (
+        <InlineNotification
+          aria-label="Close notification"
+          kind="error"
+          onClose={() => setError(dateFieldName, null)}
```

Comment:
@vasharma05 I think closing the notification, should not remove the error on the field, given that the invalid value will still be existing on the form.

Reply:
@vasharma05 have you checked this out.
Thanks